source/SIL.AppBuilder.Portal/src/routes/(authenticated)/tasks/[product_id]/+page.server.ts

```diff
@@ -0,0 +1,100 @@
+import type { PageServerLoad } from './$types';
+import { prisma } from 'sil.appbuilder.portal.common';
+
+type Fields = {
+  ownerName?: string;           //Product.Project.Owner.Name
+  ownerEmail?: string;          //Product.Project.Owner.Email
+  projectName: string;          //Product.Project.Name
+  projectDescription: string;   //Product.Project.Description
+  storeDescription?: string;    //Product.Store.Description
+  listingLanguageCode?: string; //Product.StoreLanguage.Name
+  projectURL?: string;          //Product.Project.WorkflowAppProjectURL
+  productDescription?: string;  //Product.ProductDefinition.Description
+  appType?: string;             //Product.ProductDefinition.ApplicationTypes.Description
+  projectLanguageCode?: string; //Product.Project.Language
+}
+
+export const load = (async ({ params, url, locals }) => {
+  const product = await prisma.products.findUnique({
+    where: {
+      Id: params.product_id
+    },
+    select: {
+      Project: {
+        select: {
+          Name: true,
+          Description: true,
+          WorkflowAppProjectUrl: true,
+          Language: true,
+          Owner: {
+            select: {
+              Name: true,
+              Email: true
+            }
+          },
+          Reviewers: {
+            select: {
+              Id: true,
+              Name: true,
+              Email: true
+            }
+          }
+        }
+      },
+      Store: {
+        select: {
+          Description: true
+        }
+      },
+      StoreLanguage: {
+        select: {
+          Name: true
+        }
+      },
+      ProductDefinition: {
+        select: {
+          Name: true,
+          ApplicationTypes: {
+            select: {
+              Description: true
+            }
+          }
+        }
+      },
+      // later: include only when workflow state needs it
+      // later: ProductArtifacts should be filtered based on Products.WorkflowBuildId = ProductArtifacts.ProductBuildId
+      // later: some forms don't need all artifacts, some just need aab
+      ProductArtifacts: {
```

Comment:
I thought we were going to move this to a separate query and get only the latest ProductArtifacts. There is no point getting all of the related ProductArtifacts.

Reply:
Earlier in the review you said I should just add comments for now

Reply:
I'll take care of separating it out real quick

Reply:
Done